src/app/components_ngrx/planner-query/planner-query.component.ts

```diff
@@ -163,30 +189,98 @@ export class PlannerQueryComponent implements OnInit, OnDestroy, AfterViewChecke
     }
   }
 
+  onInputKeyPress(event: KeyboardEvent) {
+    let keycode = event.keyCode ? event.keyCode : event.which;
+    if (keycode === UP_ARROW || keycode === DOWN_ARROW) {
+      event.preventDefault();
+    }
+  }
 
-  fetchWorkItemForQuery(event: KeyboardEvent, query: string) {
+  fetchWorkItemForQuery(event: KeyboardEvent, query: string, cursorPosition: number) {
     let keycode = event.keyCode ? event.keyCode : event.which;
     let queryParams = cloneDeep(this.route.snapshot.queryParams);
-    if (keycode === 13 && query !== '') {
-      if (queryParams.hasOwnProperty('prevq')) {
-        this.router.navigate([], {
-          relativeTo: this.route,
-          queryParams: {
-             q : query,
-             prevq: queryParams.prevq
-            }
-        });
+    // If Enter pressed
+    if (this.isSuggestionDropdownOpen) {
+      if (keycode === 13) {
```

Comment:
Just because the menu is opened, doesn't mean we should add a suggestion on enter. It should only do it if there is a menu item selected.